server/channels/app/web_broadcast_hooks.go

```diff
@@ -82,11 +82,8 @@ func usePostedAckHook(message *model.WebSocketEvent, postedUserId string, channe
 }
 
 func (h *postedAckBroadcastHook) Process(msg *platform.HookedWebSocketEvent, webConn *platform.WebConn, args map[string]any) error {
-	// Add if we have mentions or followers
-	// This works since we currently do have an order for broadcast hooks, but this probably should be reworked going forward
-	if msg.Get("followers") != nil || msg.Get("mentions") != nil {
-		msg.Add("should_ack", true)
-		incrementWebsocketCounter(webConn)
+	// Don't ACK mobile app websocket events at this time, we may revisit this when we can add this to mobile app
```

Comment:
We might want to - as this is for the websocket connection that's there while you're actively using the mobile app. It likely doesn't account for a lot of the notification traffic, and isn't likely as needed to "notify" you since you're already actively using the app, but for completely I think it's worth it.